### PR TITLE
V2.0.0 changes: Dynamic version of dependencies

### PR DESCRIPTION
**Pull request by shtomar-adb**
## Description

Updated RN Target wrapper and Sample app to fetch ACP Mobile dependencies dynamically and remove hard coded version of these dependencies.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: sample/ACPTargetSampleApp/Makefile [hunk 1]
```diff
@@ -0,0 +1,6 @@
+sync:
+	(cd ../../ && rm adobe-react-native-acpuserprofile-2.0.0.tgz || true)
```
Comment (yangyansong-adbe) [inline sample/ACPTargetSampleApp/Makefile]: `adobe-react-native-acpuserprofile-2.0.0.tgz` -> `adobe-react-native-acptarget-2.0.0.tgz`

File: sample/ACPTargetSampleApp/Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+	(cd ../../ && npm pack)
+	(rm -rf node_modules/@adobe/react-native-acpuserprofile || true)
```
Comment (yangyansong-adbe) [inline sample/ACPTargetSampleApp/Makefile]: acpuserprofile -> acptarget

**Review by yangyansong-adbe — APPROVED** [review]
Just one comment, looks good otherwise

File: sample/ACPTargetSampleApp/Makefile (hunk 1 above)
Comment (shtomar-adb) [reply sample/ACPTargetSampleApp/Makefile]: Done

File: sample/ACPTargetSampleApp/Makefile (hunk 2 above)
Comment (shtomar-adb) [reply sample/ACPTargetSampleApp/Makefile]: Done

**Pull request merged**